Tasks/AzureIoTEdgeV2/util.ts

```diff
@@ -88,8 +88,9 @@ export default class Util {
     if (tl.osType() === Constants.osTypeLinux) {
       cmds = [
```

Comment:
>cmds [](start = 6, length = 4)

Do you know if this change will also address this issue, customers are able to work around using this updated script:
[https://github.com/microsoft/azure-pipelines-tasks/issues/14167](https://github.com/microsoft/azure-pipelines-tasks/issues/14167)
sudo apt install python3-pip
sudo pip3 install --upgrade pip==20.3.4
sudo pip3 install --upgrade cryptography
sudo pip3 install --upgrade iotedgedev
iotedgedev --version

Reply:
I've only tested the script below on 18.04 linux and 16.04 linux, I can give these a try and test out 20.04 today